client/python/samples/README.md

```diff
@@ -54,6 +60,7 @@ usage: get_grpc_model_status.py [-h] [--grpc_address GRPC_ADDRESS]
                                 [--grpc_port GRPC_PORT]
                                 [--model_name MODEL_NAME]
                                 [--model_version MODEL_VERSION]
+                                [--timeout TIMEOUT]
```

Comment:
**This comment applies to all the samples.** 
Can we merge grpc_address and grpc_port and make user pass service_url like:
script.py --service_url localhost:9000 --model_name resnet